
class TriggerError(Exception):
    pass

class ValidationError(Exception):
    pass


